src/main/java/io/hops/hopsworks/expat/migrations/featurestore/statistics/ExpatFeatureDescriptiveStatistics.java

```diff
@@ -55,6 +62,21 @@ public class ExpatFeatureDescriptiveStatistics {
   public String extendedStatistics;
   
   public static Collection<ExpatFeatureDescriptiveStatistics> parseStatisticsJsonString(String content) {
+    if (Strings.isNullOrEmpty(content)) {
+      LOGGER.info(String.format("[parseStatisticsJsonString] file content is null or empty"));
+      return null;
+    }
+    JSONObject jsonContent;
+    try {
+      jsonContent = new JSONObject(content);
```

Comment:
Arguably everything should be put inside the `try...catch` because later you parse another JSON object. Normally it should have failed to parse the encapsulating object if something is wrong but again better safe than sorry

Reply:
Makes sense. Fixed.